packages/contracts-bedrock/test/setup/CommonTest.sol

```diff
@@ -28,13 +29,16 @@ contract CommonTest is Test, Setup, Events {
 
         Setup.setUp();
 
-        // Override the plasma config after the deploy script initialized the config
+        // Override the config after the deploy script initialized the config
         if (usePlasmaOverride) {
             deploy.cfg().setUsePlasma(true);
         }
         if (useFaultProofs) {
             deploy.cfg().setUseFaultProofs(true);
         }
+        if (customGasToken != address(0)) {
```

Comment:
should we also check to make sure it's not `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` here?

Reply:
What does this do in practice? its legal to set it to that per the specs

Reply:
You can use `addr(e)` as the gas paying token which opts into the feature with ether - its a strange semantic and side effect of having 2 addrs mean use ether

Reply:
I thought that `customGasToken` was set from deploy config, so was just trying to guard against potential user error there—but it looks like `customGasToken` will also be the zero address by the time this line executes, unless overriden by whoever inherits from it?

Reply:
I have added this check upstream of here